packages/styletron-react/src/index.js

```diff
@@ -1,5 +1,5 @@
 // @flow
-
+/* eslint-env browser */
```

Comment:
Just making note of this for myself.

So it seems `__BROWSER__ && window` is disallowed by the lint rule currently? I'll need to update it so that is allowed.